pkg/pillar/cmd/zedmanager/zedmanager.go

```diff
@@ -11,11 +11,12 @@ import (
 	"encoding/json"
 	"flag"
 	"fmt"
-	"github.com/lf-edge/eve/pkg/pillar/hypervisor"
 	"os"
 	"sort"
 	"time"
 
+	"github.com/lf-edge/eve/pkg/pillar/hypervisor"
+
```

Comment:
Extra empty line, probably added by mistake.

Reply:
Nice catch, thank you. Fixed.